src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
+        this.operationId = operationId;
+    }
+
+    public APIServerException message(String message) {
+        this.message = message;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public APIServerException errorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getErrorCode() {
+        return errorCode;
+    }
+
+    public void setErrorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+    }
+
+    public APIServerException errorClass(String errorClass) {
+        this.errorClass = errorClass;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorClass() {
+        return errorClass;
+    }
+
+    public void setErrorClass(String errorClass) {
+        this.errorClass = errorClass;
+    }
+
+    public APIServerException statusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public APIServerException errorSource(String errorSource) {
+        this.errorSource = errorSource;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorSource() {
+        return errorSource;
+    }
+
+    public void setErrorSource(String errorSource) {
+        this.errorSource = errorSource;
+    }
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        APIServerException apIServerException = (APIServerException) o;
+        return Objects.equals(this.operationId, apIServerException.operationId) && Objects.equals(this.message,
+                apIServerException.message) && Objects.equals(this.errorCode,
+                apIServerException.errorCode) && Objects.equals(this.errorClass,
+                apIServerException.errorClass) && Objects.equals(this.statusCode,
+                apIServerException.statusCode) && Objects.equals(this.errorSource, apIServerException.errorSource);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(operationId, message, errorCode, errorClass, statusCode, errorSource);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class APIServerException {\n");
+        sb
+                .append("    operationId: ")
+                .append(toIndentedString(operationId))
+                .append("\n");
+        sb
```

Comment:
Isn't it better to update the code generator to not break the statement to multiple lines? I think the generated method body is longer than needed.

Reply:
The code is actually not generated by our code generator but Swagger Codogen. So I guess no?

Reply:
yeah, I didn't noticed it's generated by swagger codegen. yeah, It's totally ok.